README.md

```diff
@@ -71,6 +71,12 @@ export FIREBASE_KEY=<FIREBASE_KEY>
 5. `set COMMAND_PREFIX=<PREFIX>`
 6. `npm run start:local`
 
+**Note**
+
+firebase_key is the firebase service key and expected key structure is json string that needs to be valid json with single quotes around the whole string and the field names/values need double quotes. An example below: 
```

Comment:
Looks good, can `firebase_key` be in capitals and inline code blocks...

```
`FIREBASE_KEY` is the firebase ...
```

Reply:
okay let me do this now